lib/sidekiq/middleware/server/freekiqs.rb

```diff
@@ -29,15 +30,15 @@ def call(worker, msg, queue)
                 raise FreekiqException, ex.message
               end
             else
-              Sidekiq.logger.info { "Out of free kiqs for #{msg['class']} job #{msg['jid']}" }
+              Sidekiq.logger.info { "Out of freekiqs for #{msg['class']} job #{msg['jid']}" }
             end
           end
           raise ex
         end
 
         def get_freekiqs_if_enabled(worker, msg, ex)
```

Comment:
nbd, but `msg` isn't used anymore in this method, is it worth removing to keep the code as simple as possible?